google-cloud-examples/src/main/java/com/google/cloud/examples/bigquery/snippets/BigQuerySnippets.java

```diff
@@ -610,6 +611,36 @@ public QueryResponse runQuery(String query) throws InterruptedException {
   }
 
   /**
+   * Example of running a query with query parameters.
+   */
+  // [TARGET query(QueryRequest)]
+  // [VARIABLE "SELECT distinct(corpus) FROM `bigquery-public-data.samples.shakespeare` where word_count > ?"]
```

Comment:
The BigQuery team would prefer we show named parameters if we had to pick just one.